em_workflows/czi/flow.py

```diff
@@ -1,28 +1,85 @@
 from pathlib import Path
+from typing import List, Dict
+import SimpleITK as sitk
 from prefect import Flow, Parameter, task
+from pytools.HedwigZarrImage import HedwigZarrImage
+from pytools.HedwigZarrImages import HedwigZarrImages
 from em_workflows.file_path import FilePath
 from em_workflows.utils import utils
-from em_workflows.config import Config
 from prefect.run_configs import LocalRun
 from .constants import VALID_CZI_INPUTS
+from .config import CZIConfig
+
+
+def rechunk_zarr(zarr_fp: Path) -> None:
+    images = HedwigZarrImages(zarr_fp, read_only=False)
+    for _, image in images.series():
+        image.rechunk(512)  # config.CHUNK_SIZE
+
+
+def gen_thumb(image: HedwigZarrImage, file_path: FilePath) -> dict:
+    sitk_image = image.extract_2d(target_size_x=300, target_size_y=300, auto_uint8=True)
```

Comment:
The `300` constant has been defined elsewhere. If they are the same, you can import it from here: 

`em_workflows/constants.py#L2`